src/routes/Project/components/Table.js

```diff
@@ -1,11 +1,16 @@
-import React from 'react'
+import React from 'react';
 
 export const Table = (props) => (
```

Comment:
thats the magic of redux, they come from the connect function call in the TableContainer file. The state is mapped to the props with the mapStateToProps function.